core/modules/filter/filter.api.php

```diff
@@ -134,28 +134,32 @@ function hook_filter_info_alter(&$info) {
  *   which should be unique and therefore prefixed with the name of the module.
  *   Each value is an associative array describing the editor, with the
  *   following elements (all are optional except as noted):
- *   - title: (required) A human readable name for the editor.
+ *   - label: (required) A human readable name for the editor.
  *   - settings callback: The name of a function that returns configuration
  *     form elements for the editor. See hook_editor_EDITOR_settings() for
  *     details.
  *   - default settings: An associative array containing default settings for
  *     the editor, to be applied when the editor has not been configured yet.
+ *   - file: The name of a file containing the editor settings callback.
+ *   - library: An associative array containing additional libraries.
  *   - js settings callback: The name of a function that returns configuration
  *     options that should be added to the page via JavaScript for use on the
  *     client side. See hook_editor_EDITOR_js_settings() for details.
  *
- * @see filter_example.module
- * @see hook_filter_info_alter()
+ * @see ckeditor.module
+ * @see hook_editor_info_alter()
  */
 function hook_editor_info() {
   $editors['myeditor'] = array(
-    'title' => t('My Editor'),
+    'label' => t('My Editor'),
     'settings callback' => '_myeditor_settings',
     'default settings' => array(
       'enable_toolbar' => TRUE,
       'toolbar_buttons' => array('bold', 'italic', 'underline', 'link', 'image'),
       'resizeable' => TRUE,
     ),
+    'file' => 'myeditor.admin.inc',
+    'library' => array('myeditor', 'backdrop.myeditor'),
```

Comment:
That might be a little less misleading, if...

`'library' => array('mymodule', 'myeditor'),`

Reply:
I can change that, though I was taking example of CKEditor module, which is in core. Please compare.

Reply:
Yes, that's like `'library' => array('ckeditor', 'backdrop.ckeditor')`

1. Module name
2. Library name